projects/netgrif-components/src/lib/data-fields/text-field/text-field.component.html

```diff
@@ -21,7 +21,7 @@
     </div>
 
     <!-- @deprecated in 4.3.0-->
-    <div *ngIf="dataField.component == undefined" [ngSwitch]="dataField.view">
+    <div *ngIf="dataField.component === undefined" [ngSwitch]="dataField.view">
```

Comment:
the check could be extracted as a class method and just called in the html

Reply:
it should be reworked as view tag is deprecated, so it should be done in another task (I think it was done on old versions, but it was never merged)